index.html

```diff
@@ -0,0 +1,198 @@
+            </div>
+        </div>
+
+        <div class="container boxes-container">
+            <div class="row d-flex justify-content-center align-items-center">
+                <div class="main-program-box col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center ">
+                    <div><img class="lang-img" src="../images/html5logo1.png" alt="htmllogo"></div>
+                    <div class=""><h2 class='h2class'>HTML Course</h2></div>
+                    <div class=""><p class="pclass">Learn HTML through various internation Speaker around the world and enjoy the best learning experience.</p></div>
+                </div>
+                <div class="main-program-box col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center ">
+                    <div><img class="lang-img" src="../images/csslogo1.png" alt="csslogo"></div>
+                    <div class=""><h2 class='h2class'>CSS Course</h2></div>
+                    <div class=""><p class="pclass">Learn CSS through various internation Speaker around the world and enjoy the best learning experience.</p></div>
+                </div>
+                <div class="main-program-box col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center ">
+                    <div><img class="lang-img" src="../images/javascriptlogo1.png" alt="htmllogo"></div>
+                    <div class=""><h2 class='h2class'>Javascript Course</h2></div>
+                    <div class=""><p class="pclass">Learn Javascript through various internation Speaker around the world and enjoy the best learning experience.</p></div>
+                </div>
+                <div class="main-program-box col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center ">
+                    <div><img class="lang-img" src="../images/reactlogo1.png" alt="htmllogo"></div>
+                    <div class=""><h2 class='h2class'>React Course</h2></div>
+                    <div class=""><p class="pclass">Learn React through various internation Speaker around the world and enjoy the best learning experience.</p></div>
+                </div>
+                <div class="main-program-box col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center ">
+                    <div><img class="lang-img" src="../images/railslogo1.png" alt="htmllogo"></div>
+                    <div class=""><h2 class='h2class'>Rails Course</h2></div>
+                    <div class=""><p class="pclass">Learn Rails through various internation Speaker around the world and enjoy the best learning experience.</p></div>
+                </div>
+                <div class=" col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center ">
+                    <button class="main-program-btn">Enroll Now in Course Batch 2023</button>
+                </div>
+                <div class=" col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center seeprogramclass">
+                    <h2>See the whole program</h2>
+                </div>
```

Comment:
- [ ] Please make sure that your `see the whole program` link is aligned correctly as in the given design guidelines. You can adjust the margin of it to get it to align correctly.
### Your project
![image](https://user-images.githubusercontent.com/92303891/218151471-32bcdf94-fe5f-4d8d-b028-ee68c81c42c7.png)
### Expected
![image](https://user-images.githubusercontent.com/92303891/218151845-842d7c9c-4c49-4c3a-babc-e098b3633f8b.png)

Reply:
![image](https://user-images.githubusercontent.com/116083031/218160748-9369fbc0-3cf7-445d-8971-aca40b21cc52.png)
Please kindly check its already aligned

Reply:
![image](https://user-images.githubusercontent.com/116083031/218160873-e529c567-a1ce-441f-979d-1d990f01099f.png)